tools/python/boutiques/localExec.py

```diff
@@ -110,10 +109,17 @@ def execute(self):
       launchDir = '${PWD}' if (self.launchDir is None) else self.launchDir
       # Run it in docker
       if conType == 'docker':
-        dcmd = 'docker run --entrypoint=/bin/bash --rm' + envString + ' -v ${PWD}:' + launchDir + ' -w ' + launchDir + ' ' + str(conImage) + ' ./' + dsname
+        # export mounts to docker string
+        docker_mounts = ""
+        for mount_string in mount_strings:
+          docker_mounts += " -v "+mount_string
```

Comment:
From this it is unclear to me exactly how we expect the strings to be passed. Intuitively I almost expect a list like the following:
`[/local/path1:/data1, /local/path2:/data2, ...]`

If that's the case, then we could just add `'${PWD}:'+launchDir` to this list, and join them in the string with a method like: `" -v ".join([dm for dm in docker_mounts])`

What do you think? The benefit in my head is that it's just internally consistent for mounts.

Reply:
yes, sure, of course!